modules/capitalize_decorator.rb

```diff
@@ -0,0 +1,6 @@
+require './base_decorator'
```

Comment:
- [ ] Kindly fix your file import here. `require` is often used for external packages like gems, while `require_relative` is used for your own local files.

  ```
  require_relative './base_decorator'
  ```
  ![image](https://user-images.githubusercontent.com/36542986/225248498-4d97ed23-fc2c-4215-b615-da6301cf7034.png)